macros/src/lib.rs

```diff
@@ -0,0 +1,96 @@
+// This file is part of radicle-link
+// <https://github.com/radicle-dev/radicle-link>
+//
+// Copyright (C) 2019-2020 The Radicle Team <dev@radicle.xyz>
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 or
+// later as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+#![feature(proc_macro_diagnostic)]
+
+use std::convert::TryFrom;
```

Comment:
Reading through https://gist.github.com/Koxiaet/8c05ebd4e0e9347eb05f265dfb7252e1#use-unambiguous-paths I can't tell if these imports should be imported with the `::` prefix. What do you think?

Reply:
No you misread that: the _generated_ code should use unambiguous paths.